include/nbl/video/utilities/CAssetConverter.h

```diff
@@ -71,6 +72,14 @@ class CAssetConverter : public core::IReferenceCounted
 		{
 			if (!params.valid())
 				return nullptr;
+		#ifndef _NBL_DEBUG
+			if (!params.optimizer)
+			{
+				using pass_e = asset::ISPIRVOptimizer::E_OPTIMIZER_PASS;
+				// shall we do others?
+				params.optimizer = core::make_smart_rectd_ptr<asset::ISPIRVOptimizer>({EOP_STRIP_DEBUG_INFO});
+			}
+		#endif
```

Comment:
@kevyuu we should move this to your `ISPIRVDebloater` (or trimmer as I'd like to call it) and make it an option to not run the SPIR-V optimizer multiple times for no reason